helsingborg-stad/app-mitt-helsingborg
# refactor: profile screen

## Explain the changes you’ve made
Refactor Profile screen to agreed format

## Explain why these changes are made
Screens and components should follow the same file structure

## Explain your solution
Created separate files for types, styling, and the component itself. Also added tests

## How to test

Concrete example:

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. Sign in with whichever user you like and go to the profile screen
4. It should work as normal

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.
